src/comp.py

```diff
@@ -23,49 +25,49 @@ def __repr__(self):
 # whose name starts with 'D':
 
 print("Starts with D:")
-r = []  # TODO
+r = [h.__repr__() for h in humans if h.name[0] is 'D']  # TODO
```

Comment:
There's no need to chain `.__repr__()` since its function is to provide a representation of the object already. `[h for h in humans if h.name[0] is 'D'] ` will give the same output. *But* I believe the exercise wants us to create a *list of names*, so the "correct" solution would be:

```py
[h.name for h in humans if h.name[0] is 'D']
```